package.json

```diff
@@ -14,8 +14,12 @@
   ],
   "dependencies": {
     "@mdi/font": "^5.0.45",
+    "babel-polyfill": "^6.26.0",
     "core-js": "^2.6.11",
+    "css-vars-ponyfill": "^2.3.2",
+    "dashify": "^2.0.0",
```

Comment:
Don't need this - this was a dependency for vue-tabulator which is not being used here.

Reply:
Removed.